persist-cli-tests/src/test/resources/test-src/output/tool_test_db_push_10/persist/entities_db_script.sql

```diff
@@ -5,8 +5,8 @@ CREATE TABLE Profile (
 	id INT NOT NULL,
 	name VARCHAR(191) NOT NULL,
 	isAdult BOOLEAN NOT NULL,
-	salary FLOAT NOT NULL,
-	age DECIMAL NOT NULL,
+	salary DOUBLE NOT NULL,
+	age DECIMAL(65,30) NOT NULL,
```

Comment:
Ballerina DECIMAL has precision upto 34. Wont this throw error for edge cases

Reply:
I checked with `0.45678987654321789087654321456786543`. It has not thrown an error. It rounds that value up to the nearest  30 scales and stores it.


We can't exceed this scale value. Its maximum value is 30. Returned the error when updating that value. 
```
ERROR: failed to read the SQL schema file. Too big scale 34 specified for column 'price'. Maximum is 30.
```

Reply:
Ack